drivers/wdm/audio/backpln/portcls/irpstream.cpp

```diff
@@ -14,6 +14,50 @@
 
 #include <debug.h>
 
+static
+KSDDKAPI
+PIRP
+NTAPI
```

Comment:
Not sure it's needed to have a locally-used-only function marked as if it was an exported public API.

Reply:
Its not, I just have no idea what those mean, I copied that function from elsewhere and modified it to suit my needs.

Reply:
You can then remove the `KSDDKAPI` and `NTAPI` thingies there.

Reply:
```suggestion
PIRP
```